### PR TITLE
Add MaciASL v1.3 cask

### PR DESCRIPTION
**Pull request by githubutilities**

### CODE REVIEW

File: Casks/maciasl.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+cask :v1 => 'maciasl' do
+  version '1.3'
+
+  if MacOS.release >= :mountain_lion
```
Comment (vitorgalvao) [inline Casks/maciasl.rb]: `if` is reversed.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in a25e9157e7312e9d2a768382472ab4b309ecf048. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

File: Casks/maciasl.rb (hunk 1 above)
Comment (githubutilities) [inline Casks/maciasl.rb]: Why it is reversed?

Comment (vitorgalvao) [inline Casks/maciasl.rb]: See the correction. `<=` is preferred. Old versions should be on top and `else` used for the newer ones.

**Pull request closed without merging**